observability/WFLY-18582_add-prometheus-endpoint-to-micrometer-extension.adoc

```diff
@@ -0,0 +1,125 @@
+---
+categories:
+ - observability
+---
+= WFLY-18582 - Add prometheus endpoint to micrometer extension
```

Comment:
Actually, this and the PR title should have '[PREVIEW]' in it. See https://github.com/wildfly/wildfly-proposals/pull/572 for an example